app/models/form/lettings/pages/person_age.rb

```diff
@@ -1,15 +1,21 @@
 class Form::Lettings::Pages::PersonAge < ::Form::Page
-  def initialize(id, hsh, subsection, person_index:)
+  def initialize(id, hsh, subsection, person_index:, is_child:)
```

Comment:
```suggestion
  def initialize(id, hsh, subsection, person_index:, person_type: "child")
```

I'd do this, personally, and if you're clever you might never have to check whether the `person_type == "child"`!